server/mux.go

```diff
@@ -121,8 +125,7 @@ func NewMux(opts MuxOpts, service Service) http.Handler {
 		return Logger(opts.Logger, auth)
 	}
 
-	compressed := gziphandler.GzipHandler(router)
```

Comment:
One alternative idea here is that I think you can disallow gzipping by setting your Accept header to: `Accept: gzip,q=0` in the request to Kapacitor according to [this](http://stackoverflow.com/questions/12321455/what-encoding-string-tells-a-web-server-not-to-send-gzip-content) Stack Overflow post. If the proxy shuts that off to Kapacitor... it'd be heavier on the backend buuut, I'm assuming the link between Chronograf and Kapacitor is generally going to be better than Chronograf to the edge and beyond.

This removal is cool too though, Assets were the heaviest parts